src/internet_identity/tests/integration/v2_api/identity_info.rs

```diff
@@ -0,0 +1,198 @@
+//! Tests that `get_identity_info` returns the correct information.
+
+use candid::Principal;
+use canister_tests::api::internet_identity as api;
```

Comment:
The goal is to keep refactoring this test until the v2 api is complete enough so that this test does not need a dependency on `api` but only `api_v2`.